package.json

```diff
@@ -6,7 +6,10 @@
   "scripts": {
     "start": "node server.js",
     "dev": "nodemon server.js & npm run dev --prefix ./client/ ",
-    "build": "npm run build --prefix ./client/"
+    "build": "npm run build --prefix ./client/",
+    "lint": "eslint ./client/**",
+    "lint:fix": "eslint ./client/** --fix",
```

Comment:
Predpokladam ze si tymto scriptom prebehol cele repo a to potom preformatovalo vsetky tieto subory? Mozme tym prebehnut aj "./**" je to tiez JavaScript. Ak si to uz nespravil, prebehol by som tym vsetky subory nech sa nam v buducnosti lintovacie zmeny nemiesaju s tymi realnymi.

Reply:
Trocha problem je to ze pedpokladam ze pre server side javascript existuju ine lintovacie pravidla. Ono v skutocnosti by sme mali mat tie projekty rozdelene na urovni package json to je mozno vec ktoru by bolo treba spravit tiez lebo toto neni good practice mat package json pre 2 defacto nezavisle projekty

Reply:
A v buducnosti by sme lintovanie mali zapojit aj do pipeline

Reply:
jj, mas pravdu, na zaciatku sme mali projekt rozdeleni do dvoch packagov jeden pre FE a druhy pre BE, ale v nejakom bode som sa zasekol na nejakom webpack setupe a mergol som ich do jedneho. a suhlas nie je to dobry practice, spravim ticket presne na toto.